apps/api/src/app.module.ts

```diff
@@ -57,6 +61,12 @@ const enterpriseImports = (): Array<Type | DynamicModule | Promise<DynamicModule
     }
   }
 
+  if (isClerkEnabled()) {
```

Comment:
Sprinkle it with some white labelling

```suggestion
  if (isEEAuthEnabled()) {
```